test/e2e/tests/confirmations/signatures/personal-sign.spec.ts

```diff
@@ -98,6 +104,60 @@ describe('Confirmation Signature - Personal Sign', function (this: Suite) {
       mockSignatureRejected,
     );
   });
+
+  it('can queue multiple personal signs and confirm', async function () {
+    await withFixtures(
+      {
+        dapp: true,
+        fixtures: new FixtureBuilder()
+          .withPermissionControllerConnectedToTestDapp()
+          .build(),
+        title: this.test?.fullTitle(),
+      },
+      async ({ driver, localNodes }) => {
+        const addresses = await localNodes[0].getAccounts();
+        const publicAddress = addresses[0].toLowerCase();
```

Comment:
nit: maybe we could use `DEFAULT_FIXTURE_ACCOUNT_LOWERCASE` from `constants.ts` so we avoid making a call and it's slightly faster?